test/e2e/search_test.go

```diff
@@ -231,7 +231,7 @@ registries = ['{{.Host}}:{{.Port}}']`
 		}
 		port := GetPort()
 		registry := podmanTest.Podman([]string{"run", "-d", "--name", "registry3",
-			"-p", fmt.Sprintf("%d:5000", port), registry,
```

Comment:
Look at all these wonderful examples of ultra-confusing code

Reply:
Thanks, Ed!  That is really confusing.